cmd/crowdsec-cli/dashboard.go

```diff
@@ -106,20 +113,42 @@ cscli dashboard setup -l 0.0.0.0 -p 443
 	}
 	cmdDashboard.AddCommand(cmdDashStart)
 
-	var remove bool
 	var cmdDashStop = &cobra.Command{
 		Use:   "stop",
 		Short: "Stops the metabase container.",
 		Long:  `Stops the metabase container using docker.`,
 		Args:  cobra.ExactArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := stopMetabase(remove); err != nil {
+			if err := stopMetabase(); err != nil {
 				log.Fatalf("Failed to stop metabase container : %s", err)
 			}
 		},
 	}
-	cmdDashStop.Flags().BoolVarP(&remove, "remove", "r", false, "remove (docker rm) container as well.")
 	cmdDashboard.AddCommand(cmdDashStop)
+
+	var cmdDashRemove = &cobra.Command{
+		Use:   "remove",
+		Short: "removes the metabase container.",
```

Comment:
I think we need a "purge" option that is going to remove docker image to tidy up our mess :)

Reply:
fixed by adding the feature in the `remove --force`